src/main/resources/database/schema.sql

```diff
@@ -67,5 +69,6 @@ CREATE TABLE IF NOT EXISTS default_transform_xslt
 CREATE INDEX ON record_log (record_id);
 CREATE INDEX ON record_error_log (record_id);
 CREATE INDEX ON record (dataset_id, europeana_id, provider_id);
+CREATE INDEX ON record (dataset_id, content_tier, metadata_tier);
```

Comment:
according to the new queries  in place of
```
CREATE INDEX ON record (dataset_id, content_tier, metadata_tier);
```
my suggestion would be
```
CREATE INDEX ON record (dataset_id, content_tier, europeana_id) 
CREATE INDEX ON record (dataset_id, metadata_tier, europeana_id)
```
what do you think?

Reply:
That is better indeed, I will put this suggestion

Reply:
It could make sense perhaps as you're not just retrieving the `europeana_id`, but also first sorting by it. So perhaps there is some benefit? I don't know, though. So I suppose you could leave it like this.